{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": []
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "cve@mitre.org",
      "ID": "CVE-2019-14575"
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "Logic issue in DxeImageVerificationHandler() for EDK II may allow an authenticated user to potentially enable escalation of privilege via local access."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": []
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "https://bugzilla.tianocore.org/show_bug.cgi?id=1608",
          "refsource": "MISC",
          "tags": [],
          "url": "https://bugzilla.tianocore.org/show_bug.cgi?id=1608"
        }
      ]
    }
  },
  "impact": {},
  "lastModifiedDate": "2020-11-23T19:33Z",
  "publishedDate": "2020-11-23T17:15Z"
}